.envrc

```diff
@@ -0,0 +1,6 @@
+layout_poetry
```

Comment:
Don't commit a `.envrc`, change this to `.envrc.example`  if you want to communicate what someone may need in order to set up their own environment: https://app.getguru.com/card/iRKjrqMT/Repository-Documentation?q=repository

Reply:
Gotcha.  Changes added.